### ISSUE TITLE
Adding type tool tip

### ISSUE THREAD

**Issue by pachacamac** [newcat/baklavajs#152]
I felt like its a bit mysterious for the user if they can't see the types when using the type plugin to check types. So I added a little tool tip to show the defined type: 

```javascript
viewPlugin.hooks.renderInterface.tap(this, (e) => {
    e.$el.title = e._props.data.type || "any"
    return e
})
```

Just in case this is useful for anyone :smile:

**Comment by newcat** [comment]
Thanks for the suggestion. I think I'll just add this to the `plugin-interface-types` directly so no hook is needed

**Comment by newcat** [comment]
Implemented in v1.10.0
